codalab/lib/bundle_cli.py

```diff
@@ -424,8 +424,13 @@ def build_parser(cls, cli):
             # Register arguments for the subcommand
             for argument in command.arguments:
                 argument_kwargs = argument.kwargs.copy()
+
                 completer = argument_kwargs.pop('completer', None)
+                arg = argument
                 argument = subparser.add_argument(*argument.args, **argument_kwargs)
+                if arg.args[0] == '--request-network':
+                    argument_kwargs['action'] = 'store_false'
+                    subparser.add_argument(*('--no-request-network',), **argument_kwargs)
```

Comment:
We should think of a better way to add this argument in. The way you've set it up, `--no-request-network` is not documented automatically in CLI reference but `--request-network` is.

Reply:
We can add this to the description to --request-network to tell the user to use --no-request-network if they want to disable network access?

Reply:
I think that ideally, we would just include --no-request-network as a new argument in the reference, and either remove the line for --request-network or put a note saying "deprecated" next to "--request-network"

![image](https://user-images.githubusercontent.com/1689183/120903889-eb0cd900-c616-11eb-813b-2a2768899ec4.png)

Reply:
@jzwang43 I think adding a --no-request-network is a bit complicated, especially because it might require us to change how the CLI reference is rendered. What do you think about this approach instead:

- remove --no-request-network
- keep --request-network, but add a custom parsing function for this argument (if needed), so that specifying `--request-network` does nothing but `--request-network false` disallows network access

Reply:
If we do `--request-network false`, I am having trouble making it backward compatible with `--request-network` without true or false. If user enters cl run --request-network xxxxx, the parser will think xxxx is for --request-network and error since the command is missing.